extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayProducer.java

```diff
@@ -37,6 +37,11 @@ public Flyway produceFlyway() {
         }
         flywayRuntimeConfig.sqlMigrationPrefix.ifPresent(configure::sqlMigrationPrefix);
         flywayRuntimeConfig.repeatableSqlMigrationPrefix.ifPresent(configure::repeatableSqlMigrationPrefix);
+
+        configure.baselineOnMigrate(flywayRuntimeConfig.baselineOnMigrate);
+        flywayRuntimeConfig.baselineVersion.ifPresent(configure::baselineVersion);
+        flywayRuntimeConfig.baselineDescription.ifPresent(configure::baselineDescription);
```

Comment:
Should these two be defined only if `flywayRuntimeConfig.baselineOnMigrate` is true?

Reply:
Yes, they are only used when Flyway creates a baseline entry in the database. That's only the case when Flyways runs against a not-empty database and when baselineOnMigrate is true. In that situation it use baselineVersion and baselineDescription.